Automattic/newspack-ads
# Update Logo

### Changes proposed in this Pull Request:

Update Newspack logo in the block picker. 

<img width="429" alt="Screen Shot 2019-08-21 at 6 37 36 PM" src="https://user-images.githubusercontent.com/1477002/63473366-2f7bb480-c443-11e9-9340-370b6f185cae.png">

### How to test the changes in this Pull Request:

1. Check out branch, run `npm run build:webpack`
2. Activate `Newspack Ads` plugin, deactivate `Newspack Blocks`, if necessary. (Both plugins will add Newspack category to block picker, so deactivating the other ensures we are testing this one only)
3. Create a post, open the block picker, scroll to the bottom.
4. Observe revised Newspack logo.
